licenses.d2iq.yaml

```diff
@@ -313,7 +313,7 @@ resources:
       - license_path: LICENSE
         ref: knative-v1.10.0
         url: https://github.com/knative/serving
-  - container_image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+  - container_image: quay.io/brancz/kube-rbac-proxy:v0.18.1
```

Comment:
Could you remove duplicate records so that we have only 1 entry for the `quay.io/brancz/kube-rbac-proxy:v0.18.1`

Reply:
Sure